pmd/pmd.xml

```diff
@@ -7,8 +7,41 @@
       Terasology PMD ruleset
   </description>
   <exclude-pattern>.*/org.terasology.protobuf/.*</exclude-pattern>
-  <rule ref="rulesets/java/basic.xml"/>
-  <rule ref="rulesets/java/braces.xml"/>
+<!-- inline <rule ref="rulesets/java/basic.xml"/>  -->
+  <rule ref="category/java/errorprone.xml/AvoidBranchingStatementAsLastInLoop" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/AvoidDecimalLiteralsInBigDecimalConstructor" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/AvoidMultipleUnaryOperators" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/AvoidUsingOctalValues" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/BrokenNullCheck" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/CheckSkipResult" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/ClassCastExceptionWithToArray" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/DontUseFloatTypeForLoopIndices" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/JumbledIncrementer" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/MisplacedNullCheck" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/OverrideBothEqualsAndHashcode" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/ReturnFromFinallyBlock" deprecated="true" />
+  <rule ref="category/java/errorprone.xml/UnconditionalIfStatement" deprecated="true" />
+
+  <rule ref="category/java/multithreading.xml/AvoidThreadGroup" deprecated="true" />
+  <rule ref="category/java/multithreading.xml/DontCallThreadRun" deprecated="true" />
+  <rule ref="category/java/multithreading.xml/DoubleCheckedLocking" deprecated="true" />
+
+  <rule ref="category/java/bestpractices.xml/AvoidUsingHardCodedIP" deprecated="true" />
+  <rule ref="category/java/bestpractices.xml/CheckResultSet" deprecated="true" />
+
+  <rule ref="category/java/codestyle.xml/ExtendsObject" deprecated="true" />
+  <rule ref="category/java/codestyle.xml/ForLoopShouldBeWhileLoop" deprecated="true" />
+
+  <rule ref="category/java/performance.xml/BigIntegerInstantiation" deprecated="true" />
+  <rule ref="category/java/performance.xml/BooleanInstantiation" deprecated="true" />
+
+  <rule ref="category/java/design.xml/CollapsibleIfStatements" deprecated="true" />
+  <rule ref="category/java/design.xml/SimplifiedTernary" deprecated="true" />
+<!--  inline <rule ref="rulesets/java/braces.xml"/>-->
+  <rule ref="category/java/codestyle.xml/ForLoopsMustUseBraces" deprecated="true" />
+  <rule ref="category/java/codestyle.xml/IfElseStmtsMustUseBraces" deprecated="true" />
+  <rule ref="category/java/codestyle.xml/IfStmtsMustUseBraces" deprecated="true" />
+  <rule ref="category/java/codestyle.xml/WhileLoopsMustUseBraces" deprecated="true" />
   <rule ref="rulesets/java/clone.xml"/>
```

Comment:
would it make sense to have some kind of "end of inline" comment?

Reply:
all is inlined, and the comments will most likely go away, i just left them to make this review easier.